documentation/data/link-type.md

```diff
@@ -5,7 +5,16 @@ uid: linkType
 # Link Data Messages
 
 
-A Link is used to create relationships between Types and instances of Types, and is defined using the pre-defined typeid `__Link` in the Data Message.
+A Link is used to create relationships between omf created resources.  Links are defined using the pre-defined typeid `__Link` in the Data Message. 
+For example:
+ - links between static instances, such as a plant to a tank, or a tank to a pump
+ - links from static instances to containers, such as a tank to a measurement
+ - links from a specific property name in a static instances to a measurement, such as a tank inflow property to a measurement
```

Comment:
use singular for the instance? => a specific property name in a static instance to a measurement

Reply:
Thanks.  Done.